helios-services/src/main/java/com/spotify/helios/servicescommon/coordination/DefaultZooKeeperClient.java

```diff
@@ -241,9 +241,14 @@ public void deleteRecursive(final String path) throws KeeperException {
       final CuratorTransactionFinal t = client.inTransaction().check().forPath(path).and();
       for (final String node : reverse(nodes))  {
         t.delete().forPath(node).and();
+        log.info("Added delete {} to transaction", node);
```

Comment:
It's a bit verbose. I'll revert this change and just run it on a personal branch.